lib/govuk_index/publishing_event_processor.rb

```diff
@@ -0,0 +1,46 @@
+require 'govuk_index/elasticsearch_presenter'
+
+module GovukIndex
+  class ValidationError < StandardError; end
+
+  class PublishingEventProcessor
+    TIMEOUT_SECONDS = 5.0
+
+    def initialize(index_name)
```

Comment:
Is `index_name` the long index name or the alias name (I think it should be the latter)